### PR TITLE
EIRH-29 EIRH-33 | @rebeccahongsf | hardcore node twig template, setup vite, and configure decanter v7

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- hardcore node twig template, setup vite, and configure decanter v7

# Review By (Date)
- when possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Create a page on your local
3. Add some paragraph blocks
4. Update `16` in the `node--16.html.twig` filename to match the node of your new page
5. Rebuild Cache and import config `drush cr ; drush ci`
6. Refresh the page you created
7. Verify that the paragraph items and the Vite + React app appears
8. Review code 🪴 
- Verify that the bundler has been configured correctly and determine if additional app compiler config is needed.

# Associated Issues and/or People
- [EIRH-29](https://stanfordits.atlassian.net/browse/EIRH-29)
- [EIRH-33](https://stanfordits.atlassian.net/browse/EIRH-33)
- [EIRH-44](https://stanfordits.atlassian.net/browse/EIRH-44)

[EIRH-29]: https://stanfordits.atlassian.net/browse/EIRH-29?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EIRH-33]: https://stanfordits.atlassian.net/browse/EIRH-33?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EIRH-44]: https://stanfordits.atlassian.net/browse/EIRH-44?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
@sherakama I'm going to add decanter to this PR. Please feel free to hold off on reviewing this for now!

File: src/eventshub/vite.config.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    rollupOptions: {
+      output: {
+        entryFileNames: 'eventshub.js',
+        chunkFileNames: 'eventshub-chunk.js',
+        assetFileNames: '[name].[ext]',
```
Comment (sherakama) [inline src/eventshub/vite.config.ts:12]: We don't have a build system on the sites stack as far as I recall so don't we need to track the dist dir as well?

Comment (rebeccahongsf) [reply src/eventshub/vite.config.ts:12]: Ah yes! That we do 😄 Let me update the `.gitignore`

File: src/eventshub/dist/index.css [hunk 2]
```diff
@@ -0,0 +1 @@
+@tailwind base;@tailwind components;@tailwind utilities;
```
Comment (sherakama) [inline src/eventshub/dist/index.css]: Shouldn't this be actual css...? Is postcss not running yet?

Comment (sherakama) [reply src/eventshub/dist/index.css]: Build vs publish thing perhaps? I have not RTFM yet on this however and could be mistaken.

Comment (rebeccahongsf) [reply src/eventshub/dist/index.css]: Postcss wasn't working due to how I defined the config. All tailwind/decanter things should be working as expected now!

Comment (sherakama) [reply src/eventshub/dist/index.css]: Huzzah!

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**